Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.h

```diff
@@ -42,7 +42,6 @@ class CLuaElementDefs : public CLuaDefs
     LUA_DECLARE(GetElementDimension);
     LUA_DECLARE(GetElementZoneName);
     LUA_DECLARE(GetElementBoundingBox);
-    LUA_DECLARE_OOP(GetElementBoundingBox);// Probably wrong, tell me how to declare the OOP version of this.
```

Comment:
No, you did it the other way around :)

Remove `LUA_DECLARE(GetElementBoundingBox)`, keep `LUA_DECLARE_OOP(GetElementBoundingBox)`.

Reply:
Ahh, alright, thanks :)